packages/create-gatsby/src/cmses.json

```diff
@@ -0,0 +1,8 @@
+{
+  "none": "No (or I'll add it later)",
+  "gatsby-source-wordpress-experimental": "WordPress",
+  "gatsby-source-contentful": "Contentful",
+  "gatsby-source-datocms": "DatoCMS",
+  "gatsby-source-sanity": "Sanity",
+  "gatsby-source-agility": "AgilityCMS"
+}
```

Comment:
No reason to block this on but shouldn't we add filesystem too?

Reply:
Do people often use that manually? 🤔

Reply:
Yes. The reason I didn;t add it initially is because I wasn't sure how we'd handle adding multiple entries as people sometimes do. But yes we do need it.